airflow/www/views.py

```diff
@@ -2280,10 +2280,13 @@ class VariableView(wwwutils.DataProfilingMixin, AirflowModelView):
     verbose_name_plural = "Variables"
     list_template = 'airflow/variable_list.html'
 
-    def hidden_field_formatter(view, context, model, name):
```

Comment:
@edgarRd  It handles fields that can possibly be hidden. FieldFormatter is too generic and not accurate in its use even if it does handle errors.